c/tskit/tables.c

```diff
@@ -4463,6 +4472,28 @@ cmp_edge(const void *a, const void *b)
     return ret;
 }
 
+static int
+cmp_migration(const void *a, const void *b)
+{
+    const migration_sort_t *ca = (const migration_sort_t *) a;
+    const migration_sort_t *cb = (const migration_sort_t *) b;
+
+    int ret = (ca->time > cb->time) - (ca->time < cb->time);
+    /* If time values are equal, sort by the source population */
+    if (ret == 0) {
+        ret = (ca->source > cb->source) - (ca->source < cb->source);
```

Comment:
Would be nice to get these branches tested - I guess it means handcrafting some tests and that's a but painful?